lib/SQL/Translator/Parser/MySQL.pm

```diff
@@ -326,7 +326,7 @@ create : CREATE PROCEDURE NAME not_delimiter "$delimiter"
 PROCEDURE : /procedure/i
     | /function/i
 
-create : CREATE or_replace(?) create_view_option(s?) /view/i NAME /as/i view_select_statement "$delimiter"
+create : CREATE or_replace(?) create_view_option(s?) /view/i NAME parens_field_list(?) /as/i view_select_statement "$delimiter"
```

Comment:
If you don't specify a field list it generates it from the column aliases, but if you specify both, the list overrides the aliases:

```
mysql> create table foo (id integer primary key);
mysql> insert into foo values (42);
mysql> create view baz (wibble) as select id as fleem from foo;
mysql> select * from baz;
+--------+
| wibble |
+--------+
|     42 |
+--------+
mysql> show create view baz\G
*************************** 1. row ***************************
                View: baz
         Create View: CREATE ALGORITHM=UNDEFINED DEFINER=`ilmari`@`%` SQL SECURITY DEFINER VIEW `baz` AS select `foo`.`id` AS `wibble` from `foo`
```